static/sched/distribits2024.xml

```diff
@@ -0,0 +1,805 @@
+        risk-managing such projects while they are under way, and a
+        software stack that helps in this task. At the same time this
+        software stack will provide an audit trail across the project
+        that is verifyable even by external scientists without access
+        to the raw data, while keeping the option for future
+        reproducability studies and replicability studies open. This
+        is achieved by utilizing git-annex and datalad in a novel way
+        to provide partial data blinding.
+        Because collecting study-relevant data is often a time- and
+        labor-intensive undertaking in the medical domain, many
+        projects are undertaken by associations that span multiple
+        hospitals, administrative domains, and often even multiple
+        states. Therefore the "Data Science Orchestrator" project also
+        implements distributed data science computations, which allow
+        to honor these existing administrative boundaries by means of
+        a federated access model, all while keeping the most sensitive
+        data in-house and exclusively in a tightly controlled
+        computation environment.
+        This work was sponsored by Deutsche Zentren für
+        Gesundheitsforschung (DZG) and BMBF.
+      </abstract>
+      <persons>
+        <person>Markus Katharina Brechtel</person>
+        <person>Philipp Kaluza</person>
+      </persons>
+    </event>
+
+    <event guid="8fadd0c8-90fa-43ba-976b-3e195ac07d93">
+      <start>14:10</start>
+      <duration>00:20</duration>
+      <title>TBA</title>
+    </event>
+
+    <event guid="f465a39d-4ed4-4609-80e0-8373e2c28739">
+      <start>14:30</start>
+      <duration>00:15</duration>
+      <title>Questions and panel discussion</title>
+      <abstract>
+        Questions and panel discussion
+      </abstract>
+    </event>
+
+    <event guid="44622511-c5c2-4ca8-9978-fa1a153b8f80">
+      <start>15:15</start>
+      <duration>00:20</duration>
+      <title>TBA</title>
+      <abstract>
+        TBA
+      </abstract>
+      <persons>
+        <person></person>
+      </persons>
+    </event>
+
+    <event guid="a81918df-a057-48b1-b016-ce31f59b02a3">
+      <start>15:35</start>
+      <duration>00:20</duration>
+      <title>TBA</title>
+      <abstract>
+        TBA
+      </abstract>
+    </event>
+
+    <event guid="42070d09-8bc1-459f-a24a-decd1382e301">
+      <start>16:00</start>
+      <duration>01:00</duration>
+      <title>Unconference</title>
+      <abstract>
+        Unconference slot
+      </abstract>
+      <persons>
+        <person></person>
+      </persons>
+    </event>
+
+    </room>
+  </day>
+
+  <day index="3" date="2024-04-06">
+    <room guid="" name="Seminar room">
+
+      <event guid="a15745bd-fa3b-4b84-afb1-ff3854b7bd09">
+        <start>08:30</start>
+        <duration>00:30</duration>
+        <title>Coffee</title>
+        <abstract>
+          Coffee time!
+        </abstract>
+      </event>
```

Comment:
We could also add a third room to this day, the "lounge" on the third floor. This is where we would have coffee